### PR TITLE
support clustersync component

### PR DESCRIPTION
**Pull request by kqzh**

### CODE REVIEW

File: exporter/type.go [hunk 1]
```diff
@@ -33,3 +42,19 @@ type (
 		ComponentType string `yaml:"componentType"`
 	}
 )
+
+func (s *StaticConfig) Validate() error {
+	for _, cluster := range s.Clusters {
+		for _, instance := range cluster.Instances {
+			switch instance.ComponentType {
+			case ComponentTypeGraphd, ComponentTypeMetad, ComponentTypeStoraged, ComponentTypeMetaListener, ComponentTypeStorageListener, ComponentTypeDrainer:
```
Comment (veezhang) [inline exporter/type.go:50]: How about add a var like `var supportedComponentType map[string]struct{}`?

**Review by veezhang — APPROVED** [review]
LGTM

File: exporter/type.go (hunk 1 above)
Comment (kqzh) [reply exporter/type.go:50]: good idea, but there won't be many instances in one exporer, so I think it's optional

Comment (veezhang) [reply exporter/type.go:50]: Yes, so I approved.

**Pull request merged**